src/Parser/Parts/PartList.cs

```diff
@@ -10,6 +10,23 @@ public PartList()
         {
         }
 
+ 
+        
+        /// <summary>
+        /// Number of <see cref="TextPart"/> with type <see cref="HoleType.Numeric"/>
+        /// </summary>
+        public int NumericPartCount { get; set; }
```

Comment:
Ah, is the case when there is no hole at all important? Could we just say it's positional?